src/lib/display.ts

```diff
@@ -0,0 +1,67 @@
+import { Display, Displayable } from '@salesforce/sfdx-scanner/lib/lib/Display';
+import {Spinner, Ux} from "@salesforce/sf-plugins-core";
+import {AnyJson} from "@salesforce/ts-types";
+
+
+export class VSCodeDisplay implements Display {
```

Comment:
This is completely a dummy implementation of Display. I do not see a need to change anything here since the display is fully handled on the VS Code side. For 5.x, we should revisit this. Do we need to pass in a Display object to the "core" package? Or is there any way for CLI to handle all the display based on the events it receives from core? I'm curious how source-deploy-retrieve package does it now.

Reply:
Did the old vs code extension implementation give real time updates from when the scanner was running? And did it forward error/warning messages to the user somehow? If so, then I don't think we can have a 100% dummy display because the display is wired up the the events and is wired up to push errors to.

Reply:
The extension has a progress bar that _very_ loosely guesses where you are in the job.